test/conftest.py

```diff
@@ -139,7 +148,7 @@ def cql_test_connection(cql, request):
 # syntax that needs to specify a DC name explicitly. For this, will have
 # a "this_dc" fixture to figure out the name of the current DC, so it can be
 # used in NetworkTopologyStrategy.
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
```

Comment:
Again, you're giving up on a performance advantage here (don't need to figure out the name of the DC in each separate test), and I didn't understand why.

Reply:
I'll try to cache this.

Reply:
Obsolete in latest. Topology doesn't need `this_dc` hack as per other comment.